dapps/templates/demo/config/communication.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
   // default applies to all environments
   default: {
-    enabled: true,
+    enabled: false,
```

Comment:
Does this mean whisper will be disabled by default in embark_demo?

Reply:
This was for testing purposes and is a left over. Will undo this change again. Good catch!